airbyte-integrations/connectors/source-yahoo-finance-price/source_yahoo_finance_price/manifest.yaml

```diff
@@ -1,312 +1,73 @@
-version: 0.50.0
+version: 0.79.1
+
 type: DeclarativeSource
+
 check:
   type: CheckStream
   stream_names:
     - price
-streams:
```

Comment:
That's also interesting. Don't get me wrong, this should work, but Builder is supposed to ref out schemas, and here it inlined it.